js/index.js

```diff
@@ -0,0 +1,71 @@
+import AwesomeBooks from './modules/books.js';
+import LocalStorage from './modules/local_storage.js';
+import CreateUI from './modules/create_ui.js';
+import CustomTime from './modules/time.js';
+
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const form = document.getElementById('form');
+
+const localS = new LocalStorage();
+const ui = new CreateUI();
+
+function getVal(title, author, id) {
+  return new AwesomeBooks(title, author, id);
+}
```

Comment:
- [ ] Just as mentioned below, please make sure to use arrow functions for all your functions.